src/main/java/org/example/jvspringbootfirstbook/repository/book/spec/AuthorSpecificationProvider.java

```diff
@@ -0,0 +1,21 @@
+package org.example.jvspringbootfirstbook.repository.book.spec;
+
+import java.util.Arrays;
+import org.example.jvspringbootfirstbook.model.Book;
+import org.example.jvspringbootfirstbook.repository.SpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthorSpecificationProvider implements SpecificationProvider<Book> {
+
+    @Override
+    public String getKey() {
```

Comment:
Let's create a constant with the field name and use it where needed in all specification providers (for example: 
`private static final String FIELD_NAME = "author";`)